<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Smart Timer App</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="timersTable">
      <column>
       <property name="text"><string>Виділити</string></property>
      </column>
      <column>
       <property name="text"><string>№</string></property>
      </column>
      <column>
       <property name="text"><string>Назва</string></property>
      </column>
      <column>
       <property name="text"><string>Час</string></property>
      </column>
      <column>
       <property name="text"><string>Статус</string></property>
      </column>
      <column>
       <property name="text"><string>Дія</string></property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text"><string>Додати</string></property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startSelectedButton">
        <property name="text"><string>Старт обрані</string></property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopSelectedButton">
        <property name="text"><string>Стоп обрані</string></property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteSelectedButton">
        <property name="text"><string>Видалити обрані</string></property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editButton">
        <property name="text"><string>Редагувати</string></property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>